src/router/index.js

```diff
@@ -7,11 +7,12 @@ import Jon from '@/components/Jon';
 import Pho from '@/components/Pho';
 
 import Header from '@/components/Header';
-
+import Weather from '@/components/Weather'
 Vue.use(Router);
 
 // Components
 Vue.component('header', Header);
+Vue.component('weather', Weather);
```

Comment:
Is this needed here?

Reply:
otherwise, LGTM

Reply:
It's registering it as a global component. So you can use it as well

Reply:
Oh okay, LGTM

Reply:
https://vuejs.org/v2/guide/components.html